src/PrismicLink.svelte

```diff
@@ -1,6 +1,7 @@
 <script lang="ts">
 	import {
 		asLinkAttrs,
+		isFilled,
```

Comment:
⚠️ #issue: I think this line can be removed since `isFilled` is no longer used. I wonder why the CI didn't catch this one.

Reply:
Fixed